lib/grape-swagger/doc_methods/path_string.rb

```diff
@@ -2,7 +2,7 @@ module GrapeSwagger
   module DocMethods
     class PathString
       class << self
-        def build(path, options = {})
+        def build(path, route_version, options = {})
```

Comment:
You can make the request an `OpenStruct` in the spec, it will actually look simpler I think. API > spec anyway.